README.md

```diff
@@ -1,8 +1,8 @@
 # VMware Carbon Black Cloud Python SDK
 
-**Latest Version:** 1.3.4
+**Latest Version:** 1.3.5
 <br>
-**Release Date:** October 12, 2021
+**Release Date:** TBD
```

Comment:
Can we pick a date? Or is this PR going to be held until security tools finish?

Reply:
Coverity & Blackduck have been reviewed this morning, both are ok.

Reply:
According to our SDK release process, the date is filled in on the new release branch once release testing is completed, and then merged back to the `develop` branch.

Reply:
Thanks for the clarification